api/src/main/java/com/theokanning/openai/completion/chat/ChatCompletionRequest.java

```diff
@@ -27,6 +27,14 @@ public class ChatCompletionRequest {
      */
     List<ChatMessage> messages;
 
+    /**
+     * Must be either 'text' or 'json_object'. <br>
+     * When specifying 'json_object' as the request format it's still necessary to instruct the model to return JSON.
+     * You may use {@link ChatResponseFormat.ResponseFormat} enum.
+     */
+    @JsonProperty("response_format")
```

Comment:
I think you can just put the class ChatResponseFormat within the request class

Reply:
I tried to follow the format that was used for other attributes.